benches/src/default_gas_costs.rs

```diff
@@ -8,7 +8,7 @@ pub fn default_gas_costs() -> GasCostsValues {
         andi: 2,
         bal: 274,
         bhei: 2,
-        bhsh: 2,
+        bhsh: 32,
```

Comment:
how has this been calculated?

Reply:
With the benchmark I modified

Reply:
I meant what steps did you follow to get this value? Asking for knowledge sharing purposes.

Reply:
I found it pretty well explained here : https://github.com/FuelLabs/fuel-core/blob/master/benches/README.md

However a little trick that green told me is to comment all the other vm opcode benches to win some time (except `noop` taht is required as a base). Maybe we should mark it in the readme.